.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+.*.sw?
```

Comment:
What's an `sw` file?

Reply:
vim swap file

Reply:
it creates `.foo.swp` while you're editing `foo`, but sometimes they get left behind and it will create another one `.foo.swo` or `.foo.swn`, etc